src/Files.App/Views/Pages/Properties.xaml.cs

```diff
@@ -58,7 +56,7 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
 			var args = e.Parameter as PropertiesPageNavigationArguments;
 			AppInstance = args.AppInstanceArgument;
 			navParameterItem = args.Item;
-			listedItem = args.Item as ListedItem;
+			var listedItem = args.Item as ListedItem;
```

Comment:
maybe to avoid further confusion we can completely get rid of this and instead of `listedItem is not null` we replace them all with `args.Item is ListedItem`?

it's not obvious because `args.Item` is of type object and can anything from a ListedItem, List<ListedItem>, a subclass, or something entirely different

Reply:
I'm not sure about this, as we would have to use `((ListedItem)args.Item)` quite a lot, which can be less confusing

Reply:
Wouldn't we have to use `(ListedItem)args.Item` quite a lot then?

Reply:
Better? [89bcf75](https://github.com/files-community/Files/pull/10886/commits/89bcf753e1cb0215ca84c088015327e52f565ed3)

Reply:
yeah, definitely better imo